packages/circuit-ui/components/NotificationToast/NotificationToast.docs.mdx

```diff
@@ -0,0 +1,137 @@
+import { Status, Props, Story } from '../../../../.storybook/components';
+import { NotificationToast } from '@sumup/circuit-ui';
+
+# NotificationToast
+
+<Status.Stable />
+
+<Story id="notification-notificationtoast--base" />
+<Props />
+
+The notification toast component non-disruptively provides quick, contextual or feedback based on the outcome of an action.
+
+## When to use it
+
+The notification toast should be used for a feedback on micro actions (i.e. successful/unsuccessful saving on the page).
+
+## Usage guidelines
+
+- To communicate a message always provide a body copy, and if needed an optional headline can be included above the body copy.
+- The notification toast floats above the content, positioned at the bottom of the screen, centralized horizontally withing the content area.
+- For a multiple stacked toasts, the new toast slides on top of the previous one. The timeout of the stacked toasts follow chronological order.
+- The auto-dismiss duration is adjustable (default minimum duration is 3000 ms).
+- Toasts can also be dismissed by clicking on a close button in the top-right corner. You need to provide closeButtonLabel.
+- **Do** Be concise, specific and short in the message.
+
+## Variants
+
+<Story id="notification-notificationtoast--variants" />
+
+### Info
+
+Use 'Info' variant for the informational, neutral messages.
+
+### Confirm
+
+Use 'Confirm' variant for the successful messages.
+
+### Notify
+
+Use 'Notify' variant for a warning messages, information to be aware of.
+
+### Alert
+
+Use 'Alert' variant for error messages.
+
+## Usage in code
+
+First, wrap your application in the `ToastProvider` which keeps track of the open toasts and ensures the accessibility of the toast.
+
+```tsx
+// _app.tsx for Next.js or App.js for CRA
+import { ToastProvider, BaseStyles } from '@sumup/circuit-ui';
+
+export default function App() {
+  return (
+      <BaseStyles />
+      <ToastProvider>{/* Your content goes here... */}</ToastProvider>
+  );
+}
+```
+
+Then, use the `useNotificationToast` hook to open a toast from a component:
+
+```tsx
+import { useNotificationToast, Button, Body } from '@sumup/circuit-ui';
+
+export function App({ name }) {
+  const { setToast } = useNotificationToast();
+
+  const handleClick = () => {
+    setToast({
+      body: 'This is a toast message',
+      closeButtonLabel: 'Close modal',
+    });
+  };
+
+  return <Button onClick={handleClick}>Open toast</Button>;
+}
+```
+
+## Accessibility
```

Comment:
One thing I'd add to the a11y section is recommendations on the timeout duration. I've read somewhere that the average American reads 240 chars (think a tweet) in 6 seconds, so we might want to recommend to use 6 seconds for a toast of roughly this length (let's do a bit more research about this)

Now that I'm thinking about this, we could even build it in, i.e. give a development-only warning if the toasts receives a body with e.g. 240 characters but only the default 3 seconds, recommending to increase the timeout duration